Source/dom/layers/SymbolInstance.js

```diff
@@ -75,6 +75,7 @@ export class SymbolInstance extends StyledLayer {
     if (this.isImmutable()) {
       return this
     }
+		this._object.ensureDetachHasUpdated()
```

Comment:
The optimal size is calculated by the detaching code, so this needs to complete before `resizeToFitContentsIfNeededNoCache` is called or it won't know the correct size.